lib/blip_ds.dart

```diff
@@ -9,6 +9,8 @@ export 'src/widgets/chat/ds_text_message_bubble.widget.dart'
     show DSTextMessageBubble;
 export 'src/widgets/chat/audio/ds_audio_message_bubble.dart'
     show DSAudioMessageBubble;
+export 'src/widgets/chat/ds_image_message_bubble.dart'
```

Comment:
Maybe you could export both DSUserAvatar and DSCachedNetworkImageView widget too. What you think about that?

Reply:
it's a good suggestion